x-pack/platform/plugins/shared/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -677,6 +693,100 @@ export class ClusterClientAdapter<
       throw err;
     }
   }
+
+  public async queryEventsBySavedObjectsSearchAfter(
```

Comment:
would it make more sense to just update the existing `queryEventsBySavedObjects` function to handle the search_after options if they're passed in?